### DISCUSSION TITLE
[Unknown Mod][Secondary Skills] Enhancement for Scouting secondary skill

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#6865] [Ideas]
Something I would really like to do, and that I would like to see added to fheroes2, is to give some boost to the Scouting secondary skill. :)

**Tweaks for Scouting secondary skill.**

Scouting gives more advanced info (right click) to the hero on map objects located in the hero's viewable area on adventure map.
All the discovered info remain visible for the rest of the game (or the entire week, ex: Water Wheel, etc.) when it's possible (as if the hero had physically visited the map objects), even if they are no longer located in the hero's viewable area (ex: Witch's Hut, Shrine, Daemon Cave, Tree of Knowledge, etc.).

**Basic level:**

- Pile of resource (ore, gems, wood, etc.): resource quantity.

- Campfire: gold and resource quantity.

- Treasure Chest: xp, gold, which artefact.

- Sea Chest: gold, which artefact.

- Flotsam: nothing, wood quantity.

- Shrine of the First Circle: spell.

- Wandering creatures level 1 to 2: number of creatures, if they will join.

- Artesian Spring: if already visited by an opponent hero for the current week.

- Oracle: distant consultation.

- Sign: distant consultation.

- Bottle: distant consultation.

**Advanced level:**

- Shipwreck Survivor: which artefact.

- Windmill: resource type, if already visited by an opponent hero for the current week.

- Magic Garden: 500 Gold, 5 Gems.

- Water Wheel: if already visited by an opponent hero for the current week.

- Artefact: if guardian, which one and how many, if secondary skill is needed (Leadership, Wisdom), cost in resource.

- Lean-To, Skeleton (Desert), Wagon: resource quantity, gold, which artefact, if already visited by an opponent hero.

- Wandering creatures level 1 to 4: number of creatures, if they will join.

- Creatures generators level 1 to 3 (Archer's House, etc.): number of creatures.

- Witch's Hut: secondary skill.

- Shrine of the Second Circle: spell.

- Tree of Knowledge: cost (free, 2000 Gold, 10 Gems).

**Expert level:**

- Wandering creatures level 1 to 6: number of creatures, if they will join.

- Creatures generators level 1 to 6 (Troll Bridge, etc.): number of creatures.

- Ancient Lamp: number of Genies.

- Graveyard: which artefact, if already visited by an opponent hero.

- Daemon Cave: number of gardians, reward, if already visited by an opponent hero.

- Shipwreck: number of Ghosts, reward, if already visited by an opponent hero.

- Derelict Ship: number of Skeletons, reward, if already visited by an opponent hero.

- Shrine of the Third Circle: spell.

- Pyramid: spell, if already visited by an opponent hero.

- Sphinx: reward, if already visited by an opponent hero.

- Jail (PoL): full hero info.

Eventually: The user can also "manually" adjust the information displayed for each level of expertise in an adjustment file (example an .xml file).

This is an option:
"skills: Scouting gives advanced info on map objects"
This can be implemented directly in fheroes2.

You can read the discussion if any about it here:
https://heroes2.forumactif.com/t943-enhancement-enhancement-for-scouting-secondary-skill#8508

**Comment by ihhub** [comment]
Making this issue as a discussion.
